### ISSUE TITLE
linkslist.app Question about saving

### ISSUE THREAD

**Issue by JamieWriter** [denno020/linkslist#7]
Hi. I created a link list using [https://www.linkslist.app/](https://www.linkslist.app/). I want to know what will prevent others who come to my list from deleting, adding, or changing the links I put there? I made this to share with people, some who are more tech-savvy than others.

**Comment by denno020** [comment]
Hi @JamieWriter, thanks for asking!

Links Lists' lists are public to anyone who knows the URL. I've done as much as I can to prevent Google from indexing the links, so people will hopefully not be able to accidentally find a list.

I have considered adding the option to lock a list, but that would lock it for everyone, including the author. This would then lead to hundreds or thousands of abandoned lists, as people will have to create a brand new list in cases where they find another item to add to the list

I am working on some premium features that will add the ability for people to lock a list, which is going to require Links List to manage passwords and authentication. I don't have an ETA on that yet, as I'm just in the beginning stages of planning.

In saying all that, I would love to be able to help out somehow.. Would the ability to "export" your list be helpful? I.e. You create a new Links List, and just before you share the URL, you export the items that are on it. Then, if the list gets modified by others over time, you can re-import the original list details, and essentially reset the list to just before it was shared. Would that help?

**Comment by JamieWriter** [comment]
Thanks for getting back to me.

I already made a copy of all the links on the list earlier today, when I
realised a simple X click could clear them.

I do see how it would lead to abandonment instead of editing to lock it.
Though, not locking it for all but the author does mean it can't be used on
a website or something.

I already sent out my link earlier, before I realized that anyone could
edit it. So, I guess whatever happens, happens. It's no big deal.

Have a good one and stay safe.
